contracts/WakalaEscrow.sol

```diff
@@ -33,7 +35,7 @@ contract WakalaEscrow  {
      /**
       * Address of the cUSD token on Alfajores: 
       */
-    address internal cUsdTokenAddress = 0x874069Fa1Eb16D44d622F2e0Ca25eeA172369bC1;
+    address internal cUsdTokenAddress;
```

Comment:
@Allen-Muhani i dont quite understand this. Where is cUsdtokenAddress now? or how does that work?
the alfjores address is gone because you dont test on alfajores? (there is still a comment talking about alfajores.

Reply:
The address will be passed as a parameter to the constructor on deployment. ie Think of it as the environment variables back at mtek. You can deploy the smart contract to any network without changing the code.

Reply:
Ah ok, got it. So you pass the address and the wakala fee.